spec/datasets/djornl/edge_type.yaml

```diff
@@ -52,3 +52,20 @@ oneOf:
     title: ATRM TF to Target LitCurated 01082020 TranscriptionFactorToGene
     description: Contains literature mined and manually curated TF regulatory interactions for A.thaliana from 1701 TFFs from PlantTFDB 2.0 and 4663 TF-associated interactions. These were manually filtered (e.g. FPs, PPI interactions removed). They then added some from other sources. Downloaded from http://atrm.cbi.pku.edu.cn/download.php
 
+  - const       : 'Metabolic_AraCyc_Gene_to_Gene'
+    title       : 'AraCyc_Gene_to_Gene'
+    description : "AraCyc.  GeneA connects to GeneB if they are both enzymes and are linked by a common substrate or product. E.g. RXNA (GeneA) → Compound1 → RXNB (GeneB). Here GeneA connects to GeneB due to Compound1.  Date accessed 03/18/2021.  All enzymatic reactions and associated genes (enzymes) and compounds were downloaded from AraCyc as smarttables, and transformed by custom R script to generate the appropriate network format linking gene to gene based on intermediary compounds.  Bringing to-spec, 322 non-standard edges were removed and 3,243 duplicates were removed."
+  - const       : 'GO'
+    title       : 'Athal_GOSemSim_BP_WangResCombined.score0.3'
+    description : "GO networks are Gene-Gene networks where an edge indicates that two genes are similar to each other based on shared GO terms.  D.Kainer used the GoSemSim R package to get pairwise similarity between all GO annotated genes.  Only GO annotations with solid evidence codes were used. The following GO codes were dropped: badcodes <- c(IEA,ISS,NAS,ND,RCA) Two Semantic Similarity measures were used: 1. Wang: a graph-based measure 2. Resnik: a simple Information Content based measure.  The average was used as the final score."
+  - const       : 'PPI_At_PPI_6merged'
+    title       : 'At_PPI_6merged'
+    description : "Made by taking the union of 6 different PPI networks: AraNet2 LC, AraNet2 HT, AraPPInet2 0.60, BIOGRID 4.3.194 physical, AtPIN, Mentha.  These 6 were all relatively good scoring with GOintersect. StringDB scored badly so was not included. dkainer (4-26-2021)"
+  - const       : 'Knockout_Oellrich_2015_supp9_Athal_gene_gene_phenosemsim'
+    title       : 'Oellrich_2015_supp9_Athal_gene-gene-phenosemsim'
+    description : "This is a network provided in the paper Oellrich 2015.  The study took the experimentally known phenotypes of over 2000 mutant konockout lines from Lloyd 2012 paper, and annotated them using the PATO trait ontology system. They then ran semantic similarity pairwise between each mutated gene resulting in a network of about 450,000 edges where an edge shows that two knocked out genes cause a phenotype that is semantically similar (0 > weight >= 1).  Uploaded by DKainer on Nov 16, 2020."
```

Comment:
Can you find the titles and DOIs for these papers and add them to the description?

> annotated them using the PATO trait ontology system

PATO = Phenotype And Trait Ontology -- you may as well just say "annotated them using the Phenotype And Trait Ontology (PATO)".

Typo: 
> 2000 mutant konockout lines